nslsii/temperature_controllers.py

```diff
@@ -38,8 +37,8 @@ def __init__(self, pv_prefix, **kwargs):
     def set(self, value):
         # check that a set is not in progress, and if not set the lock.
         if not self._set_lock.acquire(blocking=False):
-            raise Exception('attempting to set {} '.format(self.name) +
-                            'while a set is in progress'.format(self.name))
```

Comment:
Ha, neat. Looks like that with `pytest.raises(Exception)` block in the tests already _was_ masking an error other than the one we were expecting. Good object lesson.

Reply:
not sure that is true, The Travis error is due to Flake8 in .versioneer.py and .ad33.py. 

P.S. I am working on the changes to the stop method (they are a little more involved than first thought).